parser.go

```diff
@@ -339,7 +347,6 @@ func parseParam(structField reflect.StructField, fieldVal reflect.Value) (
 	paramName = tagParamParts[0]
 
 	ret := paramSetField{
-		typ:  describeType(fieldVal),
```

Comment:
At this point `fieldVal` might still be nil if the application used xtype, but did not provided the instance. A value will be created bellow, so moving the call to describe bellow to points where it is guaranteed that a value is present.